### PR TITLE
[Feature:Autograding] Fixed a typo in the workflow file

### PR DESCRIPTION
**Pull request by ION606**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?

There is a duplicate `Docker Build and Push` workflow

### What is the new behavior?
There will no longer be a duplicate `Docker Build and Push` workflow

### CODE REVIEW

File: .github/workflows/docker-build-pr.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Docker Build and Push
+name: Docker Build and PR
```
Comment (cjreed121) [inline .github/workflows/docker-build-pr.yml:1]: Change it to just `Docker Build` or `Docker Build PR`. `Build and PR` doesn't make sense.

**Review by cjreed121 — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-build-pr.yml (hunk 1 above)
Comment (ION606) [reply .github/workflows/docker-build-pr.yml]: build and pull request?

**Review by cjreed121 — APPROVED** [review]
https://github.com/Submitty/DockerImagesRPI mirrors the same workflows as this repo. Would you also be able to make a PR there for this too?

**Comment by ION606** [comment]
> https://github.com/Submitty/DockerImagesRPI mirrors the same workflows as this repo. Would you also be able to make a PR there for this too?

sure

**Pull request merged**